storybook/twig/components/media/media-image/index.js

```diff
@@ -0,0 +1,29 @@
+import { LitElement, html, customElement } from 'lit-element';
```

Comment:
Any particular reason for using JS for this? The browser by default will choose the right image to use. This won't work in IE, but as this is for a Drupal community, I doubt any one will be using it. ;)

Reply:
It doesn't refresh/change between small, medium, and large for me... so I added a resize event to try to debug and figure out why. Event runs but it still always loads the smallest image.

Similar codepen versions:
- `<picture>` version: https://codepen.io/dan2k3k4/pen/abbeORV
- `<img srcset>` version: https://codepen.io/dan2k3k4/pen/qBBedey

The `<picture>` version works with resizing the content window part but the `<img srcset>` version always loads 1200x1200 for me which is weird since the Drupal CH implementation always loads the smallest instead of the biggest (or loads whatever is in the `src` attribute...).

Reply:
What you have in the example is not 100% correct. Try: https://jsbin.com/pusukibuke/edit?html,output. If you're using Chrome it will select the largest image and use that even when you make the screen smaller as the larger image is cached. Why would it need to show the smaller image. If you make the screen smaller and try you will see it change to the larger image when you make the screen larger. Try Firefox, it will change images when making the screen smaller.

Could you please also export the config for the responsive images? I need to test my changes locally.